config/check_pr_title.py

```diff
@@ -14,7 +14,7 @@ def is_matching_name(pr_name: str) -> bool:
     if not pr_name:
         print('Your Pull Request title does not confirm to the template.')
         print('Template: Laboratory work #1, Name Surname - 19FPL1\n')
-        return False
+        return True
```

Comment:
Nice move. Try to pass this check in without disabling configs... Return to the previous state...